cmake/Modules/setHOST.cmake

```diff
@@ -30,6 +30,10 @@ macro( setHOST )
   if( NOT HOST-WCOSS_C )# don't overwrite if we are on luna 
     string(REGEX MATCH "slogin" HOST-WCOSS_C ${HOSTNAME}  )
   endif()
+  string(REGEX MATCH "alogin01" HOST-Acorn ${HOSTNAME}  )
```

Comment:
If it can be more general, it will be perfect.

Reply:
Unfortunately, this section can't be made more general.  The GSI uses the hostname for the various machines to identify what machine we are using.  Since there are two login nodes, we need to specify both potential hostnames.  Additional logic is required to ensure that CMake doesn't remove HOST-Acorn if we are using login node 1.